.github/workflows/ci.yaml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - "Jenkinsfile"
+      - ".git**"
+  pull_request:
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: '1.19.0'
+          cache: true
+
+      - name: Setup MongoDB
+        run: |
+          make start_mongo
+
+      - name: Build
+        run: |
+          make clean build
```

Comment:
Is the `clean` target of any practical use here, given that it's running with a freshly checked-out tree?

Reply:
Good catch - removed it!